roles/container-engine/containerd/templates/apt_preferences.d/debian_containerd.j2

```diff
@@ -0,0 +1,3 @@
+Package: {{ containerd_package }}

```

Comment:
Looks like a leftover from other PRs you merged into your PR. These vars are not used.

Reply:
Removed.